vendor/engines/secure_rooms/db/migrate/20170409125104_update_events.rb

```diff
@@ -0,0 +1,10 @@
+class UpdateEvents < ActiveRecord::Migration
+
+  def change
+    change_column :secure_rooms_events, :card_reader_id, :integer, null: false
+    change_column :secure_rooms_events, :user_id, :integer, null: false
+    add_column :secure_rooms_events, :account_id, :integer
+    add_foreign_key :secure_rooms_events, :accounts
```

Comment:
I just ran this migration on oracle, and we definitely do need to add the index ourselves.